test/pycheck/temporary_table_test.py

```diff
@@ -0,0 +1,27 @@
+from .utils import Cursor, PG_MAJOR_VERSION
+
+import pytest
+import psycopg.errors
+
+
+@pytest.mark.skipif(
+    "PG_MAJOR_VERSION < 15", reason="SET ACCESS METHOD was introduced in PG15"
+)
+def test_temporary_table_set_access_method(cur: Cursor):
```

Comment:
Sorry for the back and forth here. But given your latest changes (which are good changes), we should actually execute this test for PG14, just not the `SET ACCESS METHOD` part. And the name of the function also doesn't reflect what this is testing anymore now.
```suggestion
def test_temporary_table_alter_table(cur: Cursor):
```

Reply:
It's okay. Under your guidance, I've learned a lot of better things